### PR TITLE
Add gen2 root path field for gen2 storage type

### PR DESCRIPTION
**Pull request by lcadzy**

### CODE REVIEW

**Comment by lcadzy** [comment]
Fix link gen2 cluster

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionJobUploadStorageGen2Card.kt [hunk 1]
```diff
@@ -38,7 +38,9 @@ class SparkSubmissionJobUploadStorageGen2Card : SparkSubmissionJobUploadStorageB
     private val storageKeyTip = "The access key of the default storage account, which can be found from HDInsight cluster storage accounts of Azure portal."
     private val storageKeyLabel = JLabel("Access Key").apply { toolTipText = storageKeyTip }
     val storageKeyField = ExpandableTextField().apply { toolTipText = storageKeyTip }
-    val gen2AccountField = JTextArea().apply { isVisible = false }
+    private val gen2RootPathTip = "e.g. https://mystorageaccount.dfs.core.windows.net/<root path>."
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionJobUploadStorageGen2Card.kt]:
```suggestion
    private val gen2RootPathTip = "e.g. https://<mystorageaccount>.dfs.core.windows.net/<root path>."
```

**Comment by lcadzy** [comment]
#2853

**Comment by lcadzy** [comment]
#2945

**Pull request merged**